src/dogequirks.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2021 The Dogecoin Core developers
+// Distributed under the MIT software license, see the accompanying
```

Comment:
I don't think you can just slap an MIT license on someone else's unlicensed code ( http://www.voidware.com/moon_phase.htm )

Reply:
Another good point, Updated the license, cleaned up the code, and added as appropriate credit as possible. There are many algorithms to calculate moon phases, i did not want to re-invent the wheel, but do not want to steel others work either.